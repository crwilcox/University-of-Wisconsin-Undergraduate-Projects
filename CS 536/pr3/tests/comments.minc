// single line
int a = 4;
int b = 3
// single line comment 
int c = 2;
/* multi line with embedded int
int d = 1
d should not be seen...
if it does it is broken*/
int e = 0;
// int f = -1; //should not be seen as a line
