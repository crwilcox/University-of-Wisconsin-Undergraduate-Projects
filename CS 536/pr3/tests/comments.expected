line 5:0 missing SEMI at 'int'
